webapi/src/test/scala/org/knora/webapi/e2e/v2/ValuesRouteV2E2ESpec.scala

```diff
@@ -968,17 +968,6 @@ class ValuesRouteV2E2ESpec extends E2ESpec {
            |  }
            |}""".stripMargin
 
-      clientTestDataCollector.addFile(
-        TestDataFileContent(
-          filePath = TestDataFilePath(
-            directoryPath = clientTestDataPath,
-            filename = "create-int-value-with-custom-UUID-request",
-            fileExtension = "json"
-          ),
-          text = jsonLDEntity
-        )
-      )
-
```

Comment:
why is this deleted?

Reply:
Interesting finding, I can't even spot it in the git blame who did it 😄

Reply:
Ok found it - @subotic why did you delete it? https://github.com/dasch-swiss/dsp-api/pull/2002/commits/5d66fa9939c86a9d5dcb95e6e31b0f1ac95817fa

Reply:
because it was in the "do not create an integer value if the custom UUID is not part of the custom IRI" test. the "create an integer value with a custom creation date" is the next test (same file name). So nothing changes by removing it.